auth-jwt-service/pom.xml

```diff
@@ -83,9 +83,17 @@
             <Bundle-Description>AuthJWTService component for project</Bundle-Description>
             <Bundle-SymbolicName>io.stargate.auth.jwt</Bundle-SymbolicName>
             <Bundle-Activator>io.stargate.auth.jwt.AuthJWTServiceActivator</Bundle-Activator>
-            <Import-Package>io.stargate.auth, org.osgi.framework</Import-Package>
+            <Import-Package><![CDATA[
+              org.osgi.framework,
+              io.stargate.core.*,
+              io.stargate.auth,
+              io.stargate.db.*,
+              org.apache.cassandra.stargate,
+              org.apache.cassandra.stargate.*,
+              org.javatuples,
+              com.datastax.oss.driver.shaded.guava.*,
```

Comment:
importing a shaded dependency seems a bit odd. Can't we add an explicit dependency in the core module and import it here (instead of the shaded version)?

Reply:
Maybe we're exposing types from that shaded guava in shared APIs?

Reply:
The basic problem is that the shaded guava classes are actually used. The import just makes it apparent. Before this PR, these classes were imported implicitly.

Reply:
I'm +1 to using original guava artifacts, but it's probably beyond the scope of this PR.

Reply:
opened #840